lib/semian/atomic_enum.rb

```diff
@@ -0,0 +1,39 @@
+require 'forwardable'
+
+module Semian
+  class AtomicEnum #:nodoc:
+    extend Forwardable
+
+    def_delegators :@integer, :execute_atomically, :transaction, :shared?, :destroy
+    private :shared?
+
+    def initialize(symbol_list, **_options)
```

Comment:
When the SysV stuff come in, they need a name to hash on, and permissions to use for the `semget` functions, etc. I was talking about how it keeps the interface the same if the non SysV structures accept the same, but just don't do anything with it. But if you give keyword arguments to an initializer that doesn't accept keyword arguments it throws an exception.